Doc/tools/extensions/availability.py

```diff
@@ -68,14 +69,17 @@ def run(self) -> list[nodes.container]:
             refwarn=True,
         )
         sep = nodes.Text(": ")
-        parsed, msgs = self.state.inline_text(self.arguments[0], self.lineno)
+        platforms = self.parse_platforms()
+        platforms_text = (
+            f"{', '.join(starmap(_print_platform, platforms.items()))}."
```

Comment:
Not sure if babel is available here, but [`format_list`](https://babel.pocoo.org/en/latest/api/lists.html) might be a better option?

Reply:
Babel is Sphinx's dependency, it is available. Thank you for the suggestion, I've added it.